### PR TITLE
Added image

### PR DESCRIPTION
**Pull request by gzamudio**
### Summary
- Added image to book details

### Screenshots
Captura de Pantalla 2019-09-11 a la(s) 16 34 12

### Trello Card
- https://trello.com/c/uhdwuZai

### CODE REVIEW

File: scss/details.scss [hunk 1]
```diff
@@ -19,3 +19,13 @@
 		}
 	}
 }
+
+.book-props {
+	display: flex;
```
Comment (BJordi) [inline scss/details.scss]: Sort indentation.

Comment (BJordi) [inline scss/details.scss]: What you can do is create a `display.scss` file under the path `src/scss/common/`. In that file you can create common classes used to display different things that are very boilerplate:
```scss
.row {
  display: flex;
  flex-direction: row;
}

.column {
  display: flex;
  flex-direction: column;
}

File: index.html [hunk 2]
```diff
@@ -9,9 +9,12 @@
   <link href="https://fonts.googleapis.com/css?family=Roboto&display=swap" rel="stylesheet">
 </head>
 <body>
-  <p class="book-prop">Título del libro: <span class="book-prop-value">(género)</span></p>
-  <p class="book-prop">Autor del libro: <span class="book-prop-value">Nombre del autor del libro</span></p>
-  <p class="book-prop">Editorial: <span class="book-prop-value">Nombre de la editorial</span></p>
-  <p class="book-prop">Año de publicación: <span class="book-prop-value">Año de publicación</span></p>
+  <img src="img/book-cover.png" alt="Book cover"/>
+  <div class="book-props">
+    <p class="book-prop">Título del libro: <span class="book-prop-value">(género)</span></p>
```
Comment (BJordi) [inline index.html]: Do as mentioned in the previous PR.

**Review by BJordi — CHANGES_REQUESTED** [review]

File: scss/common.scss [hunk 3]
```diff
@@ -0,0 +1,9 @@
+.row {
+	display: flex;
```
Comment (nvescio) [inline scss/common.scss]: Fix indentation here

File: scss/common.scss [hunk 4]
```diff
@@ -0,0 +1,9 @@
...
+	flex-direction: row;
+}
+
+.column {
+	display: flex;
+	flex-direction: column;
+}
```
Comment (nvescio) [inline scss/common.scss]: Add an empty line here to avoid this error.

File: scss/details.scss [hunk 5]
```diff
@@ -27,3 +26,7 @@
     }
   }
 }
+
+img {
```
Comment (nvescio) [inline scss/details.scss]: Try not to style `img` tag, it's better to add a class for the image.

**Review by nvescio — CHANGES_REQUESTED** [review]

File: scss/details.scss [hunk 6]
```diff
@@ -1,3 +1,7 @@
+.book-img {
+	width: 33%;
```
Comment (BJordi) [inline scss/details.scss]: Fix indentation.

File: scss/styles.scss [hunk 7]
```diff
@@ -1,6 +1,5 @@
 body {
-    display: flex;
-    flex-direction: column;
-    font-family: 'Roboto', sans-serif;
-    margin: 0;
+  font-family: 'Roboto', sans-serif;
+  margin: auto;
```
Comment (BJordi) [inline scss/styles.scss]: `margin: 0;`

**Review by BJordi — CHANGES_REQUESTED** [review]

**Review by nvescio — APPROVED** [review]

**Pull request merged**